cmd/atlas/atlas.go

```diff
@@ -80,8 +102,6 @@ func createConfigFromMongoCLIConfig() {
 	}
 	defer in.Close()
 
-	_, _ = fmt.Fprintf(os.Stderr, `Atlas CLI has found an existing MongoDB CLI configuration file, copying its content to: %s
```

Comment:
why are we deleting this now?

Reply:
instead of saying this, we're doing the question. Otherwise we'll say in the end `AtlasCLI has copied your MongoCLI configuration to: %s` in line 128.